includes/class-newspack-newsletters-editor.php

```diff
@@ -145,6 +146,21 @@ public static function strip_block_patterns( $should_load_remote ) {
 		return $should_load_remote;
 	}
 
+	/**
+	 * Replace the Title's placeholder.
+	 *
+	 * @param string $should_replace_title Whether to replace the placeholder text.
```

Comment:
```suggestion
	 * @param string $title_placeholder Placeholder text for the title.
```

IMO the variable name `$should_replace_title` and its description imply a boolean instead of a string.

Reply:
Note that the other instances of `$should_replace_title` should also be replaced by `$title_placeholder`

Reply:
Thank you for this @dkoo. Fixed in https://github.com/Automattic/newspack-newsletters/pull/735/commits/168ba2ce2a17f200366f1021cbc48012252dd832